spec/index.bs

```diff
@@ -1280,12 +1280,15 @@ If any of the following conditions are violated:
 
 <dfn>sampled texture validation</dfn>: There must be {{GPULimits/maxSampledTexturesPerShaderStage|GPULimits.maxSampledTexturesPerShaderStage}} or
     fewer |bindingDescriptor|s of type {{GPUBindingType/sampled-texture}} visible on each shader stage in |descriptor|.
+    |bindingDescriptor|.{{GPUBindGroupLayoutBinding/hasDynamicOffset}} should be `false`.
```

Comment:
"should" should be "must" (to match the "must" just above) for all 3.

We will need to go through and improve our RFC2119 compliance at some point anyway though.

Reply:
Done. Thanks for your review, @kainino0x